plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -22,37 +22,43 @@ def to_hash(verification: false)
 
         case kind
         when 'HorizontalPodAutoscaler'
-          set_hpa_scale_target_name
+          modify_init_container do
+            set_hpa_scale_target_name
+          end
         when *Kubernetes::RoleConfigFile::SERVICE_KINDS
-          set_service_name
-          prefix_service_cluster_ip
-          set_service_blue_green if blue_green_color
+          modify_init_container do
+            set_service_name
+            prefix_service_cluster_ip
+            set_service_blue_green if blue_green_color
+          end
         when *Kubernetes::RoleConfigFile::PRIMARY_KINDS
-          if kind != 'Pod'
-            set_rc_unique_label_key
-            set_history_limit
+          modify_init_container do
+            if kind != 'Pod'
+              set_rc_unique_label_key
+              set_history_limit
+            end
+
+            set_replica_target unless ['DaemonSet', 'Pod'].include?(kind)
+
+            make_stateful_set_match_service if kind == 'StatefulSet'
+            set_pre_stop if kind == 'Deployment'
+
+            set_name
+            set_contact_info
+            set_spec_template_metadata
+            set_docker_image unless verification
+            set_resource_usage
+            set_env
+            set_secrets
+            set_image_pull_secrets
+            set_resource_blue_green if blue_green_color
           end
-
-          set_replica_target unless ['DaemonSet', 'Pod'].include?(kind)
-
-          make_stateful_set_match_service if kind == 'StatefulSet'
-          set_pre_stop if kind == 'Deployment'
-
-          set_name
-          set_contact_info
-          set_spec_template_metadata
-          set_docker_image unless verification
-          set_resource_usage
-          set_env
-          set_secrets
-          set_image_pull_secrets
-          set_resource_blue_green if blue_green_color
         end
-
-        hash = template
-        Rails.logger.info "Created Kubernetes hash: #{hash.to_json}"
-        hash
       end
+
+      hash = template
+      Rails.logger.info "Created Kubernetes hash: #{hash.to_json}"
```

Comment:
don't need these big hashes in logs ... they are stored in the release-doc anyway